### DISCUSSION TITLE
Unable to log in to https://developer.mend.io/

### DISCUSSION THREAD

**Discussion by eriksw** [renovatebot/renovate#24371] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

The login page for the hosted app at https://developer.mend.io/ shows, but clicking the "Sign In" button does nothing.

The impact of this problem is that it's stopping me from viewing run logs. (I'm trying to figure out why Renovate isn't updating a private docker image reference.)

I use uBlock Origin but there's only one resource it's blocking and there's no apparent reason why that would be a requirement.

There are no js console errors triggered by clicking the "Sign In" button.

The blocked network request is for `https://ipv4.icanhazip.com/`. Even if it's not related to the Sign In button not working, why is the app reaching out to not one but two different "what is my IP" services? 😖 

### Relevant debug logs

I am using Google Chrome `Version 116.0.5845.179 (Official Build) (arm64)` 

This is the console log after loading https://developer.mend.io/ and clicking the Sign In button several times:

```
Error with Permissions-Policy header: Origin trial controlled feature not enabled: 'run-ad-auction'.
Error with Permissions-Policy header: Origin trial controlled feature not enabled: 'join-ad-interest-group'.
Error with Permissions-Policy header: Origin trial controlled feature not enabled: 'browsing-topics'.
346-00bfceabfd06d74d.js:89     GET https://ipv4.icanhazip.com/ net::ERR_BLOCKED_BY_CLIENT
(anonymous) @ 346-00bfceabfd06d74d.js:89
A @ 346-00bfceabfd06d74d.js:89
(anonymous) @ 346-00bfceabfd06d74d.js:89
B @ 346-00bfceabfd06d74d.js:89
C @ 346-00bfceabfd06d74d.js:89
(anonymous) @ 8-08e4e49b927004fa.js:1
u @ main-1ab6d2778152c4f4.js:1
(anonymous) @ main-1ab6d2778152c4f4.js:1
a.<computed> @ main-1ab6d2778152c4f4.js:1
d @ 943-5c6bbb8edbc4469a.js:6
e @ 943-5c6bbb8edbc4469a.js:6
(anonymous) @ 943-5c6bbb8edbc4469a.js:6
(anonymous) @ 943-5c6bbb8edbc4469a.js:6
$ @ 8-08e4e49b927004fa.js:1
(anonymous) @ 8-08e4e49b927004fa.js:1
he @ framework-ae4f43955bfa5ddc.js:11
im @ framework-ae4f43955bfa5ddc.js:11
h8 @ framework-ae4f43955bfa5ddc.js:11
eP @ framework-ae4f43955bfa5ddc.js:11
il @ framework-ae4f43955bfa5ddc.js:11
ik @ framework-ae4f43955bfa5ddc.js:11
h3 @ framework-ae4f43955bfa5ddc.js:11
A @ framework-ae4f43955bfa5ddc.js:35
g @ framework-ae4f43955bfa5ddc.js:35
```

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment, marked as answer]
Please try clearing the Local Storage for the site and retrying

**Reply by eriksw** [reply]
That worked! (Curious why, but I'm no longer blocked by this.)

**Reply by rarkins** [reply]
We are trying to work out why too! Unfortunately it cannot be reproduced on demand
